Microsoft.AVS.VMFS/Microsoft.AVS.VMFS.psm1

```diff
@@ -1,3 +1,5 @@
+using module Microsoft.AVS.Management
```

Comment:
this needs to use the syntax that specifies exact required version.

Reply:
Sounds good I've updated this to include the RequiredVersion